scripts/pre_commit_hook.py

```diff
@@ -139,13 +136,15 @@ def main():
         sys.exit(0)
 
     python_utils.PRINT('Running pre-commit check for package-lock.json ...')
-    if _does_diff_include_package_lock_file_and_no_package_file():
+    if _does_diff_include_package_lock_file():
         python_utils.PRINT(
-            'This commit only changes package-lock.json without '
-            'any change in package.json. Therefore changes in '
-            'package-lock.json will be automatically reverted.')
-        python_utils.PRINT('Reverting changes in package-lock.json ...')
-        _revert_changes_in_package_lock_file()
+            'Oppia utilizies Yarn to manage npm packages. Please use Yarn '
+            'to modify packages.'
+        )
+        python_utils.PRINT('Removing package-lock.json ...')
+        _remove_package_lock_file()
+        python_utils.PRINT('COMMIT ABORTED')
```

Comment:
If we are doing this then we probably shouldn't remove the file automatically? Perhaps show a message saying "please delete package-lock.json, revert the changes in package.json, and {{an elaboration of 'use yarn'}}"?

Reply:
Done! I still kept the `COMMIT ABORTED` message since git commit fails quietly with no indication that the commit didn't go through when the check returns a non-zero code.